### PR TITLE
updating projects

### PR DESCRIPTION
**Pull request by beatrizcp87**
I'm just checking about one project, don't merge it yet.

### CODE REVIEW

**Comment by mergulhao** [comment]
@beatrizcp87 ?

**Comment by mergulhao** [comment]
@marcioc0sta can you check this with @beatrizcp87 and close this tomorrow?

**Comment by marcioc0sta** [comment]
@mergulhao I talked to @beatrizcp87 it's okay to merge, i'll update the branch now...

**Comment by marcioc0sta** [comment]
@mergulhao branch updated

File: _posts/team-members/en/2013-09-11-beatriz-correia.html [hunk 1]
```diff
@@ -36,8 +36,7 @@
 - cliqx
 - qstao
 - midiab2w
-- triagem-auditiva
-- worx-cloud
```
Comment (mergulhao) [inline _posts/team-members/en/2013-09-11-beatriz-correia.html]: @marcioc0sta why did you removed ``worx-cloud``?

Comment (biacpassos) [inline _posts/team-members/en/2013-09-11-beatriz-correia.html]: He did ask me about it, and i dindt find the screen of the project.
Em 26/01/2015 20:08, "Sylvestre Mergulhão" <notifications@github.com>
escreveu:

In _posts/team-members/en/2013-09-11-beatriz-correia.html
<https://github.com/Helabs/helabs.github.com/pull/515#discussion_r23568899>:

> @@ -36,8 +36,7 @@
>  - cliqx
>  - qstao
>  - midiab2w
> -- triagem-auditiva
> -- worx-cloud

@marcioc0sta <https://github.com/marcioc0sta> why did you removed worx-cloud
?

—
Reply to this email directly or view it on GitHub
<https://github.com/Helabs/helabs.github.com/pull/515/files#r23568899>.

**Comment by mergulhao** [comment]
@marcioc0sta Take care with the minor details. Projects are in different order, look:

sample1

File: _posts/team-members/en/2013-09-11-beatriz-correia.html (hunk 1 above)
Comment (mergulhao) [inline _posts/team-members/en/2013-09-11-beatriz-correia.html]: Ok. We can look for it in the Basecamp.

**Comment by mikaelcarrara** [comment]
@biacpassos pode dar merge nessa?

**Comment by beatrizcp87** [comment]
@mc tinha esquecido que tinha esse aqui, meu outro pr tá mais atualizado, foi mal.

**Pull request closed without merging**